hub/@hash/shuffle.json

```diff
@@ -0,0 +1,7 @@
+{
+  "repository": "https://github.com/hashintel/hash.git",
```

Comment:
I think the `examples` field in the package.json of this block should be changed from:
```TypeScript
    "examples": [
      [
        "Alice",
        "Bob",
        "Charlie"
      ]
    ],
```
to 
```TypeScript
    "examples": [
      {
        "items": [
          "Alice",
          "Bob",
          "Charlie"
        ]
      }
    ],
```

Reply:
Otherwise the array is interpreted as an object, meaning the block doesn't correctly render the example, and the "Block Properties" displays misleading fields that don't match the block schema:

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/42802102/181796242-d1654584-7829-42d8-85e9-fc07b3a51538.png">